CMakeLists.txt

```diff
@@ -33,7 +34,9 @@ if (NOT WIN32)
     find_package(Fluidsynth)
     find_package(Freetype)
     find_package(Fontconfig)
-    find_package(ICU OPTIONAL_COMPONENTS i18n lx)
+    if (NOT APPLE)
+      find_package(ICU OPTIONAL_COMPONENTS i18n lx)
```

Comment:
Have you also checked the various source lists for icu-specific files that should be excluded?

Reply:
Well, I'm not _that_ much into how OpenTTD actually works, but these two seemed safe.